src/main/kotlin/models/Archive.kt

```diff
@@ -0,0 +1,6 @@
+package models
+
+class Archive(val name: String) {
+    var notes: MutableList<Note> = mutableListOf<Note>()
```

Comment:
```suggestion
    val notes: MutableList<Note> = mutableListOf<Note>()
```
Достаточно того, что лист мьютабельный, переменную мьютабельной делать нет необходимости